app/src/main/kotlin/at/bitfire/davdroid/di/CoroutineDispatchersModule.kt

```diff
@@ -54,20 +49,13 @@ class CoroutineDispatchersModule {
      * A dispatcher for background sync operations. They're not run on [ioDispatcher] because there can
```

Comment:
I think this is good. But we should update the whole kdoc to reflect the change.

"They're not run on [ioDispatcher] [...]" is wrong, no? Should it not be something like: 

> They're run on [ioDispatcher] with a limited number (number of cores) of threads, because there can be many long-blocking operations at the same time, which should never block other I/O operations like database access for the UI.

Reply:
Ah, that's just me not seeing straight. [ioDispatcher] clearly does not refer to Dispatchers.IO, but the provide annotated method above  :+1: